example/Makefile.inc

```diff
@@ -1,6 +1,6 @@
 include $(top_srcdir)/platform/@with_platform@/Makefile.inc
 LIB   = $(top_builddir)/lib
-LDADD = $(LIB)/libodp-linux.la $(LIB)/libodphelper.la
+LDADD = $(LIB)/lib$(ODP_LIB_STR).la $(LIB)/libodphelper.la
```

Comment:
This change is according to odp-dpdk.git since we are migrating it to cloud-dev. The library name is libodp-dpdk.la. But we can change the lib name accordingly once we start making updates inside the cloud development.